### ISSUE TITLE
learning rate problem

### ISSUE THREAD

**Issue by 522545707** [thstkdgus35/EDSR-PyTorch#92]
Hi,at first,thanks for your terrific work.
I meet a problem when I want to change the learning rate:
in your option.py, I change the line #106--109

parser.add_argument('--decay', type=str, default='200',
                    help='learning rate decay type')
parser.add_argument('--gamma', type=float, default=0.5,
                    help='learning rate decay factor for step decay')

transfer --decay to '10'  --gamma to '0.9'

what I want to achieve is the learning rate decay to 90% every 10 epoch
however when I change the parameters, the learning rate just change once when epoch 10 ,when epoch 20,30,...the learning rate  doesn't change,seems like the decay just once not every 10 epoch,I don't know why,could you please helping me with this problem?

thanks for your time.

**Comment by thstkdgus35** [comment]
Hello.

You can achieve your desired scheduling strategy by using ``--decay 10-20-30-40-50`` something like that.

Otherwise, please refer to [this lines](https://github.com/thstkdgus35/EDSR-PyTorch/blob/b13e310f762e3753f4f63f0fbd5be8a9410ec4ea/src/utility.py#L202-L205) and the [official documents](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate) to achieve what you want.

Thank you!
